futures-executor/src/local_pool.rs

```diff
@@ -205,6 +205,12 @@ impl LocalPool {
     }
 }
 
+impl Default for LocalPool {
```

Comment:
I could see arguments for not implementing `Default` for `LocalPool`, if so I think having an explicit `allow` for it is good documentation that it has been considered and decided against.

Reply:
I can't think of any such arguments ^^' Could you list some?

Reply:
Mostly that a `LocalPool` isn’t a data structure, so being able to create a default in a generic context isn’t that useful. IMO it’s still worth having the impl anyway though.